snmp/datadog_checks/snmp/data/profiles/generic-router.yaml

```diff
@@ -3,10 +3,10 @@
 
 extends:
```

Comment:
Note: we could also rename this to `generic.yaml` (with `generic-router.yaml` kept as an alias for backward compat)

Reply:
How do you do an alias?

Reply:
Ok, I created a new file `generic-network` with the contents of `generic-router` and then made `generic-router` extend it, is that right?

Reply:
I think the best option would be to keep the `_generic-router-if.yaml` and only have
```yaml
extends:
  - _generic-if.yaml
```
(same for the other files)

See https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/profiles/_base_cisco.yaml

However, if customers are not using the `generic-router` files at all, better remove them

Reply:
I think the point of `_` prefixed profiles is the're _private_ but I don't know if that's documented anywhere

Reply:
I can't find documentation about the _private_ stuff either. I thought the main meaning of the `_` prefix was that the files can't directly be used as a profile since they don't have a `sysobjectid`, so I may have missed some info here.

However I don't know why someone wouldn't be able to do:

`mycustomprofile.yaml`
```yaml
extends:
  - _base.yaml
  - _generic-router-if.yaml

metrics:
  -  MIB: AWESOME-MIB
     symbol:
       OID: 1.1.1.1.1
       name: coolname
```

Reply:
Yes, that's right. This would be a breaking change @naaltman what do you think?

Reply:
I know of at least once case (wayfair) where they're overriding the _generic-router-if and adding to it, how would this effect them? 

Also can you explain a little more what the breaking change is here/how current users of the integration might be effected by it?

Reply:
The breaking change is that the file `_generic-router-if` does no longer exist but we could keep it